modules/virt-example-ethernet-nncp.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-ethernet-nncp_{context}"]
+= Example: Ethernet interface NodeNetworkConfigurationPolicy
+
+Configure an ethernet interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+
+The following YAML file is an example of a manifest for an ethernet interface.
+It includes samples values that you must replace with your own information.
+
+[source,yaml]
+----
+
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: eth1-policy <1>
+spec:
+  nodeSelector: <2>
+    kubernetes.io/hostname: <node01> <3>
+  desiredState:
+    interfaces:
+    - name: eth1 <4>
+      description: Configuring eth1 on node01 <5>
+      type: ethernet <6>
+      state: up <7>
+      ipv4:
+        dhcp: true <8>
+        enabled: true <9>
+----
+<1> Name of the Policy.
```

Comment:
Question that I have asked you before, but that might have a new answer now that the formatting guidelines have been updated: should Policy be capitalized throughout this document? If yes, should it be in backticks?

Reply:
Good question. Not in backticks, as 'Policy' is only part of the full object name. I'm reluctant to drop it down to 'policy' though because we also have other policies that impact VMs. Maybe using the acronym (NNCP) makes the most sense but runs into the danger of blurring with NNCE, and that distinction should remain clear. 
I'd like to keep capitalised for now as I think that makes the most sense for now, but I've raised an item for our next meeting for addressing the decapitalisation of objects in our docs en masse.